### PR TITLE
writing unit test for redux orders #582

### PR DESCRIPTION
**Pull request by khrystynaDm1307**
## Description

write test for redux orders

#### Screenshots
image

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/redux/order/tests/order.actions.test.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    const result = {
+      type: SET_IS_ORDER_CREATED,
+      payload: true
+    };
+
+    expect(setIsOrderCreated(true)).toEqual(result);
+  });
+
+  it('should add payment method', () => {
+    const paymentMethod = 'CASH';
```
Comment (AndriyLymych) [inline src/redux/order/tests/order.actions.test.js:62]: please, replace this const to variables

File: src/redux/order/tests/order.actions.test.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    const result = {
+      type: GET_PAID_ORDER,
+      payload: paidOrder
+    };
+
+    expect(getPaidOrder(paidOrder)).toEqual(result);
+  });
+
+  it('should return fondy data', () => {
+    const paymentMethod = 'CASH';
```
Comment (AndriyLymych) [inline src/redux/order/tests/order.actions.test.js:82]: same

File: src/redux/order/tests/order.sagas.test.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      .provide([[matchers.call.fn(addOrder), orderExample]])
+      .put(setOrderLoading(true))
+      .put(setOrder(orderExample))
+      .put(setIsOrderCreated(true))
+      .put(setOrderLoading(false))
+      .run();
+  });
+
+  it('fetching order error', () => {
+    const message = 'error';
```
Comment (AndriyLymych) [inline src/redux/order/tests/order.sagas.test.js:43]: same

File: src/redux/order/tests/order.sagas.test.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    expectSaga(handleOrderError, message)
+      .put(setOrderLoading(false))
+      .put(setError(message))
+      .put(push('/error-page'))
```
Comment (AndriyLymych) [inline src/redux/order/tests/order.sagas.test.js:47]: same

**Comment by AndriyLymych** [comment]
please, fix CI
image

**Comment by AndriyLymych** [comment]
please, link issue
image

**Comment by AndriyLymych** [comment]
please, add screen your passed tests and fill checkboxes
image

**Comment by AndriyLymych** [comment]
please, add screenshot with passed tests from your terminal
image

**Review by AndriyLymych — APPROVED** [review]

**Review by MMashivskij — APPROVED** [review]

**Pull request merged**